decentraland/unity-renderer
# refactor: move web3 outside of "shared"

## What does this PR change?

Refactors code out of `shared` (which should be only for sagas-based code)

## How to test the changes?

1. Launch the explorer
2. Ensure it connects correctly to the world
3. Go to a Web3 Gallery (such as the one in genesis plaza and open the NFT Dialogue -- ensure the owner is displayed correctly
